pkgs/tools/security/pcsclite/default.nix

```diff
@@ -58,8 +57,8 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoreconfHook pkg-config perl ];
 
   buildInputs = [ python3 ]
-    ++ lib.optionals stdenv.isLinux [ dbus polkit systemd ]
-    ++ lib.optionals stdenv.isDarwin [ IOKit ];
+                ++ lib.optionals stdenv.isLinux [ dbus polkit systemd ]
+                ++ lib.optionals stdenv.isDarwin [ IOKit ];
```

Comment:
We usually do not intent lines like this in nixpkgs.

Reply:
Fixed, sorry, shouldn't have let that through.